charts/openshift-metering/templates/hive/hive-configmap.yaml

```diff
@@ -71,18 +71,14 @@ data:
         <name>javax.jdo.option.ConnectionDriverName</name>
         <value>{{ .Values.hive.spec.config.db.driver }}</value>
       </property>
-{{- if .Values.hive.spec.config.db.username }}
       <property>
         <name>javax.jdo.option.ConnectionUserName</name>
         <value>{{ .Values.hive.spec.config.db.username }}</value>
       </property>
-{{- end }}
-{{- if .Values.hive.spec.config.db.password }}
       <property>
         <name>javax.jdo.option.ConnectionPassword</name>
         <value>{{ .Values.hive.spec.config.db.password }}</value>
       </property>
-{{- end }}
```

Comment:
revert this

Reply:
The way it was in the beginning, if you inspected the yaml on hive-configmap in the cluster - the username and pw xml properties did not show at all.
There are 2 options to get it to display:
- Remove the helm templating like above or 
- Aligning the conditional nesting in the xml tags like this:

      <property>
        <name>javax.jdo.option.ConnectionUserName</name>
{{- if .Values.hive.spec.config.db.username }}
        <value>{{ .Values.hive.spec.config.db.username }}</value>
{{- end }}

which is not what would happen if I revert. A revert would put back the removed lines.

Reply:
This commit shows a way to keep the conditional but also have it display
e83cfc4ff85d54b95372706cbf66a9bfea458589